ui/tests/acceptance/clients-list-test.js

```diff
@@ -55,6 +55,38 @@ module('Acceptance | clients list', function(hooks) {
     assert.equal(nodeRow.allocations, allocations.length, '# Allocations');
   });
 
+  test('each client record should show running allocations', async function(assert) {
+    server.createList('agent', 1);
+
+    const node = server.create('node', {
+      modifyIndex: 4,
+      status: 'ready',
+      schedulingEligibility: 'eligible',
+      drain: false,
+    });
+
+    server.create('job', { createAllocations: false });
+
+    const running = server.createList('allocation', 2, { clientStatus: 'running' });
+    server.createList('allocation', 3, { clientStatus: 'pending' });
+    server.createList('allocation', 10, { clientStatus: 'complete' });
+
+    await ClientsList.visit();
+
+    const nodeRow = ClientsList.nodes.objectAt(0);
+
+    assert.equal(nodeRow.id, node.id.split('-')[0], 'ID');
+    assert.equal(nodeRow.name, node.name, 'Name');
+    assert.equal(
+      nodeRow.compositeStatus.text,
+      'ready',
+      'Combined status, draining, and eligbility'
+    );
+    assert.equal(nodeRow.address, node.httpAddr);
+    assert.equal(nodeRow.datacenter, node.datacenter, 'Datacenter');
+    assert.equal(nodeRow.allocations, running.length, '# Allocations');
+  });
```

Comment:
I think you can cut out many of these assertions because they’re already covered by the previous test. It looks like the composite status and allocations assertions are the only ones that differ.